src/components/ChatEntry.js

```diff
@@ -1,22 +1,49 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+
+  const onLikeButtonClick = () => {
+    const updatedChatEntry = {
+      id: props.id,
+      sender: props.sender,
+      body: props.body,
+      timeStamp: props.timeStamp,
+      liked: !props.liked,
+    };
+    props.onUpdateChatData(updatedChatEntry);
+  };
+
+  const heart = props.liked ? '❤️' : '🤍';
+
+  let chatStyle = 'chat-entry local'
+  if (props.id %2 !== 0) {
+    chatStyle = 'chat-entry remote'
+  }
```

Comment:
Great work. The optional alignment feature would require us to be able to pick whether to use the `local` or `remote` class here. If the `App` made note of the sender of the first message, and passed that down as a prop, such as `localSender`, then we could use that value and the value of the current message's `sender` to determine whether this was a `local` or `remote` message.